bip-0002.mediawiki

```diff
@@ -133,31 +133,46 @@ Will BIP comments be censored or limited to particular participants/"experts"?
 
 ==BIP licensing==
 
-New BIPs may be accepted with the following licenses:
-
 ===Specification===
 
+New BIPs may be accepted with the following licenses. Each new BIP must identify at least one acceptable license in its preamble. The License header in the preamble must be placed after the Created header. Each license must be referenced by their respecitve abbreviation given below.
```

Comment:
I don't think we should start defining ordering of headers here.

Also, it is undefined how to interpret multiple licenses (OR vs AND; probably want OR).